listComment.js

```diff
@@ -0,0 +1,51 @@
+import Sequelize from 'sequelize';
+import Comment from './db/models/comment';
+import User from './db/models/user';
+import db from './config/db';
+
+export async function main(event, context, callback) {
+  // Do not wait for sequelize connection to close
+  context.callbackWaitsForEmptyEventLoop = false;
+
+  const headers = {
+    'Access-Control-Allow-Origin': '*',
+    'Access-Control-Allow-Credentials': true,
+  };
+
+  Comment.init(db);
+  User.init(db);
+
+  Comment.belongsTo(User, {foreignKey: 'created_by'});
+  User.hasMany(Comment, {foreignKey: 'id'});
+
+  let response;
+  let statusCode;
+
+  try {
+    const result = await Comment.findAll({
```

Comment:
What's the use case here? We will probably never want to list all the comments in the DB. I think it would be better if we just had a `GET` endpoint for all the comments on a certain video.

I think something like `GET /videos/{id}/comments` would be more useful.

Reply:
True that would make more sense. 
Right now for each video, all the comments are queried and then filtered on the client side for the specific video